src/web/aws-cloudwatch/StepReview.jsx

```diff
@@ -1,68 +1,110 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
-import { Button, Col, Row } from 'react-bootstrap';
 import styled from '@emotion/styled';
+import { Col, Row } from 'react-bootstrap';
 import { Link } from 'react-router';
 
-import { Input } from 'components/bootstrap';
 import Routes from 'routing/Routes';
+import { Input } from 'components/bootstrap';
 
-import DEFAULT_VALUES from './default_values';
+import { FormDataContext } from './context/FormData';
+import { LogOutputContext } from './context/LogOutput';
+import FormWrap from '../common/FormWrap';
 
-const StepReview = ({ values, onSubmit, onEditClick, logOutput }) => {
-  const defaultOutput = (key, enabled = true) => {
-    if (!enabled) {
-      return (
-        <React.Fragment>
-          {DEFAULT_VALUES[key]} <small>(default)</small>
-        </React.Fragment>
-      );
-    }
+const format = (numString) => {
+  return Number(numString).toLocaleString('en-US');
```

Comment:
Maybe we should leave out the locales so we are defaulting to the browser/os setting?